### DISCUSSION TITLE
Export wallpaper from WallpaperEngine in mp4 format

### DISCUSSION THREAD

**Discussion by doryan04** [Almamu/linux-wallpaperengine#170] [General]
Hello! Is it possible to export the wallpaper to mp4 or gif video format with this program? There is a variable "videopath" in the program code and I would like to know where the video is stored (if at all).

**Comment by UltraBlackLinux** [comment]
it's pretty easy. Find the directory of the workshop creation and there should be some kind of archive. There usually is a video file in there

**Comment by Almamu** [comment]
There's no support for recording video of the backgrounds. Depending on the background you're using (if it's of the type "video") you can have access to the original video file. This folder usually lives somewhere inside the steam folder -> steamapps -> workshop -> 431960 -> background ID.

**Reply by doryan04** [reply]
Okay, thanks!
